arelle/plugin/validate/DBA/rules/th.py

```diff
@@ -16,6 +16,28 @@
 
 _: TypeGetText
 
+
+@validation(
+    hook=ValidationHook.XBRL_FINALLY,
+)
+def rule_th05 (
+        pluginData: PluginValidationDataExtension,
+        val: ValidateXbrl,
+        *args: Any,
+        **kwargs: Any,
+) -> Iterable[Validation]:
+    """
+    DBA.TH05: Contexts should not contain segments or unclosed periods
```

Comment:
`or unclosed periods`

Does that mean this should also be raised for contexts with forever periods?